Wheat tidyup

HamishBrownPFR opened the pull request:
working on #7623

Moved all met files into met folder
Moved all excel files into data folder
Moved all csv files with model inputs into input folder
Deleted some files that are not used.

HamishBrownPFR commented:
@hol353 @par456 This PR just moves all the met and observed files into specific folders to help tidy up the wheat tests.  Pleas review and comment or merge if happy

hut104 commented:
These changes will likely be changed again as we redo the build system. Perhaps talk with our software team.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Hamish Brown ***@***.***>
Sent: Friday, January 31, 2025 8:48:55 AM
To: APSIMInitiative/ApsimX ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [APSIMInitiative/ApsimX] Wheat tidyup (PR #9634)


working on #7623<https://github.com/APSIMInitiative/ApsimX/issues/7623>

Moved all met files into met folder
Moved all excel files into data folder
Moved all csv files with model inputs into input folder
Deleted some files that are not used.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/APSIMInitiative/ApsimX/pull/9634

Commit Summary

  *   ccc4605<https://github.com/APSIMInitiative/ApsimX/pull/9634/commits/ccc4605478122241dd3bf29f5c5db6f76cd72e2e> Move files into folders
  *   b2e5e77<https://github.com/APSIMInitiative/ApsimX/pull/9634/commits/b2e5e777161522044ae0b1f297c5057bf50bd716> Merge branch 'master' of https://github.com/APSIMInitiative/ApsimX into WheatTidyup

File Changes

(276 files<https://github.com/APSIMInitiative/ApsimX/pull/9634/files>)

  *   D Tests/Validation/Wheat/APS14TimeSeries.txt<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-34690e706c4469785bef6030ee7993e1d2b72720d897f72b8944b74a10373026> (14)
  *   D Tests/Validation/Wheat/APS26TimeSeries.txt<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-b21fffd200a61119a797a25f00e20bc65a4c53eccf3ed329e01831b3641efe18> (35)
  *   D Tests/Validation/Wheat/APS6Timeseries.txt<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-49794f2cfea661b063164046a14b298142b6d3b9f0c95da91c39a3e3a8fa459e> (46)
  *   D Tests/Validation/Wheat/KonyaMetData.xlsx<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-95ab3e6d904414d4529127a31fd0072906039c1ba8c7d772568ffe5536e2cb45> (0)
  *   D Tests/Validation/Wheat/NPIField2019_ZadokStage.csv<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-40adc16e6d1d938d1538a023567a59dabff57b6bbe5f944984f11d11b9137311> (8232)
  *   D Tests/Validation/Wheat/NPIValidation.7z<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-a40f2ae0f28834625a2eac3ca7fd2f28fa9ea992183502a083d5498e07a9ee3e> (0)
  *   D Tests/Validation/Wheat/RainShelterBiomass.txt<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-6ad1869d796e863ea2c86dce715801ef6b5a32d7849ee7bddaed06971b121fb1> (194)
  *   M Tests/Validation/Wheat/Wheat.apsimx<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-1f09629aaa0bb7d9ca77fc17bdd8f697e622f8c852648af84e6a3ff19029341f> (560)
  *   R Tests/Validation/Wheat/data/2020Results.xlsx<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-41c0309e66e49890cf21b98b47afdb46211d73c2418a9302af74b657a19ae0de> (0)
  *   R Tests/Validation/Wheat/data/Gatton2014_2016.xlsx<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-cb4a084655942361bb45d0c8ddcc9b0e650fa61f18eaaa30148f1d4777e2881c> (0)
  *   R Tests/Validation/Wheat/data/LaTrobeCE_Obs.xlsx<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-8b0e418a40d1d396627fb7c5ebd232c96b692b8cf56d41d9845dcc31fda3b39a> (0)
  *   R Tests/Validation/Wheat/data/Lincoln2021.xlsx<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-931c179093b7c1f13e525b5e9662c2641a5469ed56ab23b20959b04afeac1c53> (0)
  *   R Tests/Validation/Wheat/data/Lincoln2023.xlsx<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-d5b5ff14611868ffda32ad3d6d309ee2f91166f71257f1f0686ea339aa0f0e90> (0)
  *   R Tests/Validation/Wheat/data/LincolnCE_Obs.xlsx<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-479a95917644cbd4e8159b719cc03546c928e515916bb18c47c019e5a856549d> (0)
  *   R Tests/Validation/Wheat/data/NPIValidation.xlsx<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-d80dcaa5c8f6b93bec7e735a6b656591929838fc338c6b8687acd208472f26df> (0)
  *   R Tests/Validation/Wheat/data/Obs_APSIMX_NPIField2019_Wheat.xlsx<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-36035d0715c5a7ac903ef00b4811b660af2d3f5cb1f25430cc0cb5bf3a6b100e> (0)
  *   R Tests/Validation/Wheat/data/Obs_APSIMX_NPIField2020_Wheat.xlsx<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-0089c8d2caca8a3b60f021eb108df835bc046e310f55b09c6e591fb2239f242c> (0)
  *   R Tests/Validation/Wheat/data/Observed.xlsx<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-31297dd35908159d4a6e998a00bb44995fa5534471a73b0af31c321348b21e00> (0)
  *   R Tests/Validation/Wheat/data/PalmerstonNorth1989.xlsx<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-38ecb32309a8db0e23a70ee0540267157e56d57c7afbe5a603a0eef698b1d853> (0)
  *   R Tests/Validation/Wheat/data/PalmerstonNorthCE_Obs.xlsx<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-6ee53ff437bf259f4bdc876baee92f82b7552d306cd0e73bf5e56535a4373c78> (0)
  *   R Tests/Validation/Wheat/inputs/CallingtonEmergenceDates.csv<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-b30c953aa742c3d0f40cebe435b614ce0b3464d73bed8126537822df022d7464> (0)
  *   R Tests/Validation/Wheat/inputs/DaleEmergenceDates.csv<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-4db742ce3a349b045982f0daa8ce1ea2a72e49d9c1a5a64be74aa1f2dc44f0b4> (0)
  *   R Tests/Validation/Wheat/inputs/DaleEmergenceDates2020.csv<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ac7e5df3786bb650ad28be53ae017ac4548965b6ad96726c2cc2341e7e16f41f> (0)
  *   R Tests/Validation/Wheat/inputs/LaTrobeEmergenceDates.csv<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-cf1d01f713bf631254ba26d1e555e3db27f10e78078e1dd2395aef76bea47e74> (0)
  *   R Tests/Validation/Wheat/inputs/UrrbraeEmergenceDates.csv<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-c9ae4701a30f6880eacc93554ca36619d3a6f3fa11124b32e688ea91a90d32f0> (0)
  *   R Tests/Validation/Wheat/inputs/WaggaWaggaEmergenceDates.csv<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-7245e8d5763158077cbbef51a67cfff3f69a5fc54383c2203f50914d5f8c7137> (0)
  *   R Tests/Validation/Wheat/inputs/WaggaWaggaEmergenceDates2020.csv<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-f43b307e054d37e1d85537678cbd84d1987c2bb2e22141c22a84d73cc15fdb89> (0)
  *   R Tests/Validation/Wheat/inputs/YanYeanEmergenceDates.csv<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-d64df9b17da3b4c75f17de9aceb721a3d61da883a27139954ca89a1340d4a476> (0)
  *   R Tests/Validation/Wheat/inputs/YanYeanEmergenceDates2020.csv<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-120c18a9d6b79207ef90d744f430be940b36542b867632e5752696951e1951a5> (0)
  *   R Tests/Validation/Wheat/met/12JuneeJames-1-100.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-3ac10acc8be8bfa9e56f5b00281d1ab72356677af96d60258dcaf03fa0dac914> (0)
  *   R Tests/Validation/Wheat/met/12JuneeJames-1-50.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-a5ceda4698b99bf71cd947314dc4ee043b615c9d620972af41491e89418fd083> (0)
  *   R Tests/Validation/Wheat/met/12JuneeJames-2-100.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-8377ffb02c5448c159b29edadb5f6e58847068085c40e3df91a78ba337a381a7> (0)
  *   R Tests/Validation/Wheat/met/12JuneeJames-2-50.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ad621e09107fb73c31f451cf868b8d2f6ece8340421f6e594e601e506f719086> (0)
  *   R Tests/Validation/Wheat/met/12JuneeJames-3-100.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-17e3d8b0e1e7bcf93a8aaa1e186e5c12aa97afd9024d17303efc6a93a61ca8ec> (0)
  *   R Tests/Validation/Wheat/met/12JuneeJames-4-100.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-94d3342baed2bf2f21cbf4e6f1863585a30d769d1f32de6be01705e0ada8e1e7> (0)
  *   R Tests/Validation/Wheat/met/14SSOW-1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-34a1f06dbc0b0498b9ed6aa2a93ec228b76ba9b5fd588237ff258f193f3ed117> (0)
  *   R Tests/Validation/Wheat/met/14SSOW-2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-65d530dba6735dc6ee54f250628abc76338a5e7c6ed159fc5cee2f81d06f3b6d> (0)
  *   R Tests/Validation/Wheat/met/14SSOW-3-V1P1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-e6ddbcc63393cd9b0cf862f1d8862b44e224742af7b4596001b5a38a3cacfafb> (0)
  *   R Tests/Validation/Wheat/met/14SSOW-3-V1P2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-6edd724143f6fd25ed3b13db9ab2897fd9a848688ce3167f8b62635ed3d454ff> (0)
  *   R Tests/Validation/Wheat/met/14SSOW-3-V2P1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-b7bc7a4b736e55a71b8f53ba2dc75b6ddcc9a17aad8c9f569da9ee31091370b2> (0)
  *   R Tests/Validation/Wheat/met/14SSOW-3-V2P2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-21bed32edaa1747da47a4c4e74ae1d3f9fb3ac1e2d2df23f4aec6cc9c04854de> (0)
  *   R Tests/Validation/Wheat/met/14SSOW-4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-cddc7183e0a47ee4559e60233d1d039c33e69d38b0e6b762787461a1a6fbe178> (0)
  *   R Tests/Validation/Wheat/met/14SSOW-5.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-39725979b5c80ac3d978d2a176a4314e2474dd4e4e30aadf5ab9d8dfd3be3ca6> (0)
  *   R Tests/Validation/Wheat/met/15SP-V1P1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-bc311a70d73056a3e2786135982bd2ce1370e705bc770c92d4ccd0dbf50ac2e8> (0)
  *   R Tests/Validation/Wheat/met/15SP-V1P2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ffec8a11d82ed5e6ec11147bd9db6eafdf92bf4726452e1ac8ea8f253e8da0ff> (0)
  *   R Tests/Validation/Wheat/met/15SP-V2P1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-709a188cb589dc1630835c231c8b561a2efd3f444cd2045e0c95fe98e0d1754c> (0)
  *   R Tests/Validation/Wheat/met/15SP-V2P2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ca902665fa11cbd5a230e2584ab436c3d2b82fa654b51e5edf2e83b431aa974f> (0)
  *   R Tests/Validation/Wheat/met/17CuryoMESW-TOS1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-4dfce0a5c2ead27d470b33b8622bddc15abdfa0f7dfedd81789a898a8091f241> (0)
  *   R Tests/Validation/Wheat/met/17CuryoMESW-TOS2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-2ceb116985031fc64a4dc457580a67be01873a76d9755d7b1b2417bf4d65a375> (0)
  *   R Tests/Validation/Wheat/met/17CuryoMESW-TOS3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-7f4a721e6ecd0cf9a348aafc288ff398124808459529219b9bbce707a6efd443> (0)
  *   R Tests/Validation/Wheat/met/17CuryoMESW-TOS4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-4f84c412b9637d85b94ef517b87104fc36922c2fcd165ac05d9e03cd84fcd1ab> (0)
  *   R Tests/Validation/Wheat/met/17HartMESW-TOS1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ecb98d0615ff98f495159b0048755d1aa240b2ea772710ef49684be2152a6a59> (0)
  *   R Tests/Validation/Wheat/met/17HartMESW-TOS2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-9ff0f54abe1e4043646d685bcc8b43c8815062108e6f23e057becb8e54d61397> (0)
  *   R Tests/Validation/Wheat/met/17HartMESW-TOS3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ec2623b43219faa2e5e951e6967779a0f015ef592a0e722074fc6869f2eb0a18> (0)
  *   R Tests/Validation/Wheat/met/17HartMESW-TOS4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-cea8f4e4ba5af1a4602e9d184cfcbb01c73c36521a0bcda46153946895a08510> (0)
  *   R Tests/Validation/Wheat/met/17LoxtonMESW-TOS1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-73ba70f09bd3fba76a4b3c4dd7e861eaa70c8b5f9c96b83bac2471dea8dce79f> (0)
  *   R Tests/Validation/Wheat/met/17LoxtonMESW-TOS2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-d669736851a74e254239222cb41635c8ec57b10c0f9da33cbf91b9eeaece45ec> (0)
  *   R Tests/Validation/Wheat/met/17LoxtonMESW-TOS3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-97fb95dc1d5460c5956b662269dcbed1ca08d702d094d9d87509a0f0d207f22f> (0)
  *   R Tests/Validation/Wheat/met/17LoxtonMESW-TOS4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-422f5406955eed5d3e1c1f18500f535c614c68438fb033cb6c79693f821ae21d> (0)
  *   R Tests/Validation/Wheat/met/17MilduraMESW-TOS1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-2aeef216793fac9b3bced5953717ba3f99efe1735332ae2e9a03bc02369b30a3> (0)
  *   R Tests/Validation/Wheat/met/17MilduraMESW-TOS2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-7ca51a4ccb7db383d5cf727e4d5895b7516df53b506b37d468f70f802f073cf5> (0)
  *   R Tests/Validation/Wheat/met/17MilduraMESW-TOS3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-3346f0fe6501e5b5189ed38379e81cf48b4de901c4a8226be74516504a02a30c> (0)
  *   R Tests/Validation/Wheat/met/17MilduraMESW-TOS4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-06f800b8e42062604e78798552c00ffdd9ed5eccaf68c3c883198aa97e466542> (0)
  *   R Tests/Validation/Wheat/met/17MinnipaMESW-TOS1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-7910214c93f84b322308f763a6ed08665b41d01932aeca32a50481423d5eb15d> (0)
  *   R Tests/Validation/Wheat/met/17MinnipaMESW-TOS2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-c8284b11630407a4faf558ec041dc058c9977805ecbad35caa4cd3020ff9be2c> (0)
  *   R Tests/Validation/Wheat/met/17MinnipaMESW-TOS3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-8eafc163e1c8863cb574bd6ea3b7bbe4ff63320a84c4ab73f4102c27acada2c1> (0)
  *   R Tests/Validation/Wheat/met/17MinnipaMESW-TOS4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-802a94dc250bdfcdcf98c9c6b19702adb2c10db00e9070e7c421ce1391808e71> (0)
  *   R Tests/Validation/Wheat/met/18HartMESW-TOS1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-d2c0cb9da97a287ff880e6f2e85e85c87c71cccb643f1c85cabcc3e6552bf409> (0)
  *   R Tests/Validation/Wheat/met/18HartMESW-TOS2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-e11a6e2d900d8854b10712f13ca56d20c444a41d6721973d470c644cee9be218> (0)
  *   R Tests/Validation/Wheat/met/18HartMESW-TOS3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-89e96c99a9bb38e7d474c9c58c62ebe748b93ff244eeef973b887092bc3379d4> (0)
  *   R Tests/Validation/Wheat/met/18HartMESW-TOS4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-43a042637a36155633b40a73032145143265da7f1f90d55253dc91e8e4f0e059> (0)
  *   R Tests/Validation/Wheat/met/18LoxtonMESW-TOS1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-62b2e9fa1f6ea50465874fcd1c8223fd2af6b3488b605d217a9e35ce6e4f33bf> (0)
  *   R Tests/Validation/Wheat/met/18LoxtonMESW-TOS2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-0e4709d429f8a20135be06b14cf857b064631889a0ca5b3d711339ddfcd146b4> (0)
  *   R Tests/Validation/Wheat/met/18LoxtonMESW-TOS3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-253b5141dbd0aa92aff08d42c27145e43930e59b39cb8169bfc6704b15279139> (0)
  *   R Tests/Validation/Wheat/met/18LoxtonMESW-TOS4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-372779ae91808a2f2830aa3a6c7381ee3516e721c205194e5a51e68fbd9d43c8> (0)
  *   R Tests/Validation/Wheat/met/18MilduraMESW-TOS1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-e9b881ddee63990efb278dd7da0fd3a52c1206800171162ceb9cff0096cb7219> (0)
  *   R Tests/Validation/Wheat/met/18MilduraMESW-TOS2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-186ae4d4cf21dcdbc8cf95520b55ede6a03c97c0aeed7f1586bcbbae25213c79> (0)
  *   R Tests/Validation/Wheat/met/18MilduraMESW-TOS3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-72452c64e7719215bef72c790908ba6030becf876328ed315ec8eb2ae1f7a7eb> (0)
  *   R Tests/Validation/Wheat/met/18MilduraMESW-TOS4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-88a95be4f7a015a85da9814351c9f81de5a14974501773dbb2f436d365997b0d> (0)
  *   R Tests/Validation/Wheat/met/2020NPI_Dale.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-9c397811fb6e7b8261301d6f348dc50cf588ae54fb075ca4a0bee783f6f691dd> (0)
  *   R Tests/Validation/Wheat/met/2020NPI_Urrbrae.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-d3fe7cfa32208057cf49790a488135be77ef94dc227fbc5545334ed39f3cf9bc> (0)
  *   R Tests/Validation/Wheat/met/2020NPI_WaggaWagga.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-d455295d667387daf6652290fca346a2323530f53ea2082d7fe9fd26de2e129c> (0)
  *   R Tests/Validation/Wheat/met/2020NPI_YanYean.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-f3613925607dc21a7cf0a865f635eec2c695f736df7d3583a6d2033813644d54> (0)
  *   R Tests/Validation/Wheat/met/APS14.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-9c0fd0f2e35b7cffd426b65a272aef2c3c2e5e13c58a05863bcc2dc0c1a74f28> (0)
  *   R Tests/Validation/Wheat/met/APS26.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-192f400a2e82e0f47dd1951c9e034e384390d3cbc989ea6b42605a48a38e0d06> (0)
  *   R Tests/Validation/Wheat/met/APS26Silo.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-10d875caf0a114def8d5193461dbbb2bd29fee568aa039198458e917067f1ce4> (0)
  *   R Tests/Validation/Wheat/met/APS6.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-e1b4fb6b5c3ac58c255437eb5bbb4b1d22fcbd95d9ca8e731192d79367db1caf> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Ardingly_1992-06-19.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-eae163112f922f5f4bc0241b6f96b46fc895b07a21f63c4a092c10a697c49024> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Badgingarra_2005-05-26.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-f9c51d91bdbb0c4868a11643baabfcf003cdaeb97b86c677d07c5e4945f4b62a> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Badgingarra_2005-05-28.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-fe6c7e91eb53c40569edc357f918f160c7c041d03b6805a42e4884aaf0154f4f> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Badjaling_2003-05-30.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-e1a148815f23ad91cf1dbe41023d1021cac247818bb20fad7177a0a2374366fc> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Chapman_1992-06-23.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-dd42041ea49222206bd754471998aea7acb40c45d9537f9ac5b0958d52725e05> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Chapman_1992-06-24.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-975ef20a76bf5753feb83ac2784e07a9c7c37534547274868e237611bf8cb683> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Chapman_1994-06-10.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-33d727c3ac7e04f1a7e7bfb16d15f5262e9ed9395d7b64ad81cbb8b62b21039c> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Gairdner_River_2004-06-09.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-a71ac4befe8d63b42abc6af9c4fe7804cac5ba5d9c5025eef972a493f172680d> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Georgina_1992-06-26.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-339281420c7a0b8092b664ccada506e970f098611fc83d5d32f6eb6cf6e901e4> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Konnongorring_1997-06-13.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-953d3d95414ad2cc8c8b3569099497ef6cf3a2a4584a47e4ae918bff63f5d1a2> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Konnongorring_1997-06-27.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-7f0342cc51833ed0a36033304c45113aea02de3700a6606fc7b04fef8bf31517> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Kumarl_1999-05-22.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-41a14bd38cc4539351851fa17663d2011a07627e1c5524e80a563bfe46e42c4a> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Kunjin_2005-05-26.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-055625d14a83e5277e7ee0f1c6b87bc553e577568908d1496a59e93d9787c5fc> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Meckering_2003-06-04.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-eec6e3dc9b1cb482e2226032ecb351ae87adb685104ceaff7e2fcbbb5503f7f8> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Merredin_1995-05-30.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-7569c044b6c571721637ccd5a4ed603f8831e42a890464da506304fd66a62b4d> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Merredin_1996-06-25.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-f7d60d1ff88cfb7e58cf3bb11f080cbbc9ee713ec8eb7fc15f8e0bc0b500c756> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Merredin_1996-07-02.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-77665acd7daed03de3502903557352626a1d881b634a33f9c7f856c26779755a> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Merredin_1998-06-19.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-6efeb8a7844313370f772943cb1f8dc5fcc963db6d0871dd2badff99142b0a49> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Merredin_2000-06-19.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-61ebb36864802f31819e9884d3baa1fa92a31aa90c3f027804c997f311ad8892> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Merredin_2002-06-10.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-313b628cdcfa7cb2d1c4199116e15a76265b9db87ac4eb8b909335d56d128707> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Merredin_2003-06-06.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-e6e7dd967b380737e37fbffb7e2715febb9ad2fb9e4ce03a7af275a3afba78a7> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Morbinning_2004-05-31.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-cc3a711a1d5ed10efca01a36b4a986232f3289c0e829e87b7934f485c79078d8> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Mt._Madden_1994-06-13.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ca3bd41f5784b84d2e3ebf7590b2435d194e36460da1c1712b0e4579a08df0cf> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Mukinbudin_2000-06-13.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-c4065870c619d4cb6feec2fc707b4f3134cd476af8429fb897c13cee8fa9e66a> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Mullewa_1993-06-04.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-46f6f5dc9b1e4bdf932dce4906cf6f59c20e1b871e82c8fbe1d4436be60fb7c7> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Mullewa_2004-05-28.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-a7fc633fe25d6e5ea869dc983479a639768466a5c855d43c3da5095f8ed343f1> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Munglinup_1994-06-08.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-3fab0d5fafad82fd1013e1de22edfa2bb01acb8be9e09a5dd72fc03c6607dc26> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Scaddan_1999-05-25.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-744e8b7ea99ebf3b2f45885e6c43345eb265e5079153097dee28f936a9a52d70> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Speddingup_2001-05-30.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-411dd2d4957b4da67ece0d6ceea95c58e136e3f148d49806d59d13b765fdb3dc> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Tammin_1999-06-09.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-84ffceb3e1afa1d056b4b2288e71611daf7ec9eb5109257afb9ddc4d3687d4cf> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Wannamal_1992-06-25.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-5e5bfc59e8455e5c9a54f080bc9f8112909181e05670cfb4a3f507e355d435f1> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Wongan_Hills_1998-06-09.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-2ca875024e52bea143a57c1c500031ba8842a536d2b619abf556e703384c293d> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Wongan_Hills_2001-06-11.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-15f0faabf1e236a3e1a532070aed4821152bf43b7a036fb4eb1b33d6ea719ffd> (0)
  *   R Tests/Validation/Wheat/met/BCVT_Wongan_Hills_2002-06-09.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-e54d4995bf40a48de7b3c7877e85d5925214847b72cca13778842a79d3bea341> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2008GE1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-77f06596fa0bc292e2de52a4096e031ea71e9930eec1bcde8b7572c381e9bfab> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2008GE2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-cf7c7403d519dc9c495a8b0599cb40f4bf2a7d62321860fed177e758676b1aa5> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2008GE3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-8d29f7da575b757c77ac0127e33808e63e98647938ee51007946cc33636d8989> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2008GE4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-40a4f56afed0b5b346ff72ed5dcfc050ca9260fd6ef629435ffa2810362b4a4a> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2008KA1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-cbe3fd724b963523a4d0516a048b4590981a15ed7b03d7d4819b75434943db73> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2008KA2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-a00054cc559bb3249cc91e4f29f1c12678877fea2d8c7dd7747b459a7d1eaf0f> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2008KA3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-c70fdb622a4efd287f45bdf1b4e4180aa573a7314bb36eae8af66327fbb27580> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2008KA4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-4f8da0a6e8d9f03ad4fb928636aa12209d41583e6c4cad95c93e44daa565c270> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2008NM2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-95a97ce17edc45d683f77d203810fb920ab03d8a583ff836ec2d338b82c1f925> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2008NM3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-2271d269127c588fd7342a06a683aa065b36fdcb83b807bd469b3246c84a5ed6> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2008NM4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-5bb3e530aec61e13725becd50481ef29fc18e3355d5cff8baf26b8798e9fb100> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2009GE1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-cc36f8cf2b676f96bb2f317161c7afd1195c1b10c8edf9b04388364be15f5f3b> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2009GE2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-98277de744cab4b3e72243cefbcbb06db74caa96ec0b3e8c23481429fcdf65c3> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2009GE3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-edfbec12da6d701b2411f72d2bb64b718555d3aa56bb08a0ec5a8b1ec4022ca7> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2009GE4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-bb9be5fc560140b29b95a7b9cf1f641b8410065b471ccb56b66eb2a82cbe1e02> (0)
  *   R Tests/Validation/Wheat/met/BTOS_2009KA3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-b4fdde14df0d669878eb49ac17e240ca9beefdff0d80e26ea624eb77b76ed394> (0)
  *   R Tests/Validation/Wheat/met/BVPT_08SPV1P1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-2f833fc4fddca8c8bdc9e9e3bc9de3689ffab0f3ce7ab2ccd741ae5a72625fce> (0)
  *   R Tests/Validation/Wheat/met/BVPT_08SPV1P2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-17070525e2783cdae2cd9ef87f019b632103758444cd6ebe68783b5c8e4afa9f> (0)
  *   R Tests/Validation/Wheat/met/BVPT_08SPV2P1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-f5b5ea820514e542cb97c51685db099028dc6af7a96729038a6b71f50d6ea33b> (0)
  *   R Tests/Validation/Wheat/met/BVPT_08SPV2P2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-3564e07be1c9ee5f2f9912b80dcd47506cffd8649607a2f29b9d716492ebe0b4> (0)
  *   R Tests/Validation/Wheat/met/BVPT_09SPV1P1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-fcd4a24f7d58ec25db37bd32b4bdda5bd9b8b8c0d44f0ed97883a2c0e28918ab> (0)
  *   R Tests/Validation/Wheat/met/BVPT_09SPV1P2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ff80cbc743e9f6ef8b5de900886f822bd451d05ed9ecd21402fdc83e2f5d45bc> (0)
  *   R Tests/Validation/Wheat/met/BVPT_09SPV2P1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-a686ef21efcc2a1be4e8dde1998a97ad9e4a907d2a8fa123e47bfd525cb4a602> (0)
  *   R Tests/Validation/Wheat/met/BVPT_09SPV2P2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-7c9632020065e2e2a8ca70b8963bde9b813ee05f9b03b8476e1ec7e208d4b97a> (0)
  *   R Tests/Validation/Wheat/met/BYIE_08HIR.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-fdf56f05e4f73b7925d198e36b393e8b1edbe9c48895fa27125ea34b4b776f6f> (0)
  *   R Tests/Validation/Wheat/met/BYIE_08KA.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-9793ef4e3004664bd95381014c1c1147914a754a02fdae97478e25af7943b72c> (0)
  *   R Tests/Validation/Wheat/met/BYIE_08NB.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-b491d2d4596575d768963ad50a1f24a083c4b71f549c785876ad0cced629d512> (0)
  *   R Tests/Validation/Wheat/met/BYIE_08RS.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-5f66eb1198927420b77586f8b943274dc388a3834e95d648f2b2344c6c8c0a37> (0)
  *   R Tests/Validation/Wheat/met/BYIE_08WH.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-6b74ecfd8ef22b7808ce358975ae899e7facd259f37d4fd856c0c2c999711528> (0)
  *   R Tests/Validation/Wheat/met/BYIE_09ER.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-f9f86dcd06bb278f3d171ac729aa710cf86f211e9e4813a2d25c27845c589002> (0)
  *   R Tests/Validation/Wheat/met/BYIE_09GN.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-d605938f9da872f8138440c29afc7fb19778837c95b7bb0f5e652fbeb6f9a57d> (0)
  *   R Tests/Validation/Wheat/met/BYIE_09HIR.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-3442dd28aac0f804fa9467f59d1dfe2e7b98dbc86dcd573e580f21223cb8dcfd> (0)
  *   R Tests/Validation/Wheat/met/BYIE_09KA.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-c429e313ce57fd1f158566ebd5f47d34a37a4d7505562d283578b30f1cc1452d> (0)
  *   R Tests/Validation/Wheat/met/BYIE_09MDL.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-b037e85d4d1361a730ce16cab8ee84e48e06947d624172cf7f67077ee66e2a92> (0)
  *   R Tests/Validation/Wheat/met/BYIE_09RS.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-1b891872d1b0e5a04e784b594a5a9b9c8eaf6a768477ad26c33e6ec45ee1c28f> (0)
  *   R Tests/Validation/Wheat/met/BYIE_09WH.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-e88caa9b1541423a3919501a8db23bed6410c35cd5e8f1d71bbc30df161fe94f> (0)
  *   R Tests/Validation/Wheat/met/Biloela1996.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-66db907f8305ecb98bd61ae6cbfa81f6ff8fa3c682ef65070df8ff732be985f9> (0)
  *   R Tests/Validation/Wheat/met/Birchip2011.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-fb57ae7dac598e44dcebc46efb28bfecae6e4b4e52c90714e15b6a657184b49d> (0)
  *   R Tests/Validation/Wheat/met/Brookstead2015-TOS1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-dce8b26c4b90abcb4340a435432e8941e44a24b1c8aa46ad77ecf2379a41030a> (0)
  *   R Tests/Validation/Wheat/met/Brookstead2015-TOS2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-e5dad01c24259246bed481fb86202583ec3fb71bc1af5219f5544f037066d56e> (0)
  *   R Tests/Validation/Wheat/met/Brookstead2015-TOS3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-b9277380cf924d3866622871b1ed6f19bac26460c96dbdb0350da894b5b98205> (0)
  *   R Tests/Validation/Wheat/met/BrusselsSouth.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-4804baf8e6a3c07a9bf870211869c722b0d89b234cded3bff056502c19562292> (0)
  *   R Tests/Validation/Wheat/met/Bungunya2012.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-053dd47e0dcf5288e19380544c6f4404be00532b73a86bba3d70459c5a9697ce> (0)
  *   R Tests/Validation/Wheat/met/Callington2019.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-93c1de5799532e62a5bd49e220f2d40534432e9f7f0e235a9f6849d9bb4f4eef> (0)
  *   R Tests/Validation/Wheat/met/Callington2019NPIfield.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-fe2010f4c7dcf7028a799755d0d6eaeaec08f4942687fa6e4d5ec2631fa0f1f8> (0)
  *   R Tests/Validation/Wheat/met/Cunderdin.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-6c253a4c8d2e82896a72c168bd6a862f155bd55380d458d35e7885a83858c3c5> (0)
  *   R Tests/Validation/Wheat/met/Emerald2015-TOS1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-cfdcc4496b8a9142e4418b7cd92bb8be30322b04d6de26bd0179b0664255e87e> (0)
  *   R Tests/Validation/Wheat/met/Emerald2015-TOS2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-f0414f57267ec6a52b09a0566bbb8264f36fd430946c74cf16505f016b3bd9f9> (0)
  *   R Tests/Validation/Wheat/met/Emerald2015-TOS3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-5f5e92913b7a88fb0f77e87c519299902d3ca0d4f71ee5481a96f0db89b9d01c> (0)
  *   R Tests/Validation/Wheat/met/Gatton1994.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-547b4c87510d63fd2d5f7742356a73b7c028dfc58417cac7691c32666e38698f> (0)
  *   R Tests/Validation/Wheat/met/Gatton2009.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-98628bf060a587e1b5d8f7ff1dd6beae5f9322a6cdbf82d27c04a1de183cb126> (0)
  *   R Tests/Validation/Wheat/met/Gatton2010.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-663ef2d4623ff619c6b811e25f3f651806ba67552f6d5013ad5add00aa702864> (0)
  *   R Tests/Validation/Wheat/met/Gatton2010Late.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-9960728a858c42ef1a1f120f7a34d65a0cb07a525829c0cbab4668b51203efbb> (0)
  *   R Tests/Validation/Wheat/met/Gatton2011.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-291639e5038cc2f8fa125a0cd6e6ecf673da254d9f0b7e2bb20858167fb4ee1a> (0)
  *   R Tests/Validation/Wheat/met/Ginninderra.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-42e9915bec29e629c2e6dac1db08d5259aeed5bb549c184930fcbdde6cbbbe54> (0)
  *   R Tests/Validation/Wheat/met/Goondiwindi1996.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-aa2b11fedc00b114f1fafc088084261bcedea973ea70827741cc514a366561cc> (0)
  *   R Tests/Validation/Wheat/met/Goondiwindi2011.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-db9384a418e4cfbaa9d9e39d2f1c8dc280da5daf92b59af71ebc184e3db78269> (0)
  *   R Tests/Validation/Wheat/met/Gorgan.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-d1ecbe084630d4c6509441ce8fb79b18dbd0c26ee06b98ffa79335330c7badc2> (0)
  *   R Tests/Validation/Wheat/met/Griffith1983.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-a1ab6c1822f5130f33b15774dc214ad910b7be0e6e825d599b320d5e387784d9> (0)
  *   R Tests/Validation/Wheat/met/HAGT_09Roseworthy-1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-5bcc57130c3b90e9d4fcfcfa3a0a9088017ca20f17dff27f1a78b415ba84ee3e> (0)
  *   R Tests/Validation/Wheat/met/HAGT_09Roseworthy-2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-94dbdf3dc35cc1b51b9136432c69764643f876c2a7b775b9ceb6bea40271904b> (0)
  *   R Tests/Validation/Wheat/met/HAGT_09Roseworthy-3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-dbdbc1834e0400cc76001f62745c44de96e5283ea76126f4d5c3219ed7a028db> (0)
  *   R Tests/Validation/Wheat/met/HAGT_09Roseworthy-4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-a985f07e9c40c23ed0a05ec903c5c76642b23ed4086d5b9bdae8be5bede5e814> (0)
  *   R Tests/Validation/Wheat/met/HAGT_09Roseworthy-5.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-a09e52e3708bc2723ed64b8ecf7360b50d4e62178847e01dffdfedc2a110da6d> (0)
  *   R Tests/Validation/Wheat/met/Hart2015-TOS1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ea2c47af8f8d58347bbc3098ebfefdb0bf58518ac6bac7e54913f13c59d37dd7> (0)
  *   R Tests/Validation/Wheat/met/Hart2015-TOS2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-4522cb4c6d3fb17b7ec779f49e7b41915aa0a7f8151c053562508000cad5653f> (0)
  *   R Tests/Validation/Wheat/met/Hart2015-TOS3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-2f1eaa5877cf75450a121de0bf3d590a7e2b640f8a0a3ffee2661114b1f03aeb> (0)
  *   R Tests/Validation/Wheat/met/Hermitage2009.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-933d2a9dd2f0878dac0b01d1329d306b8a55a37f1b179b56afcec07e75079a9a> (0)
  *   R Tests/Validation/Wheat/met/Hermitage2011.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-5b2437216ce0b51a88b0915fdee3f6a8323b51b54e1cc637fa20bbc1c069fc18> (0)
  *   R Tests/Validation/Wheat/met/Inverleigh2013-TOS1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-c1bbb40761f9912896951d4df99847e6fb0ef95778f6bb4e7aaed7252b038250> (0)
  *   R Tests/Validation/Wheat/met/Inverleigh2013-TOS2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-83452d2d4a24c7bb0bff8ca3f0e2c681516a651aa4467aa60c95b8d1ac40627b> (0)
  *   R Tests/Validation/Wheat/met/Inverleigh2013-TOS3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-c818e8010ea515084d0385cc1a9d3f0bb34989fff2808d382e5ff9a7148bc9bc> (0)
  *   R Tests/Validation/Wheat/met/Inverleigh2013-TOS4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-5b0686571791f4b4dfe03eb7f185c0667404227fe4c01352a01d4ab6057cb809> (0)
  *   R Tests/Validation/Wheat/met/Jamma.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-213368a4f1cabce85fc35703301d3b796ed35cd0aaa19873753874f8c47e9ebb> (0)
  *   R Tests/Validation/Wheat/met/Konya.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-158b0c84092765c845bfab365e4c1f2bc9388f341af1c994d43fda67a6b7c828> (0)
  *   R Tests/Validation/Wheat/met/LN.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-2a9f7345d977a1c8b5735d3c3f4562cfb786999d3040aeac5c6dce16def9be1a> (0)
  *   R Tests/Validation/Wheat/met/LV.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ce78a50bd7e7a689d25c6ae8fa1f28256e2478aaaab554e0c424f74ed4e37e22> (0)
  *   R Tests/Validation/Wheat/met/Leeston.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ca3745fbb451d6affa2c0f243eee5916d201b4e710264e726079d951f2399d43> (0)
  *   R Tests/Validation/Wheat/met/Lincoln9192.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-64d025631c5d8ece29ae098f665c6da7729fd2a868098e9acbd19f8a2e93d046> (0)
  *   R Tests/Validation/Wheat/met/LincolnUni.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-c73a3cc0836ef862ff1b8f3d82613f70ead7775284a21f9d5835c287a03b8448> (0)
  *   R Tests/Validation/Wheat/met/Lincoln_23.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-fac2e9cdabbc3479a943667c94b703773ea0c0a92a350a5a1d3177c7ea47a7c6> (0)
  *   R Tests/Validation/Wheat/met/MARA1992-1993.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-7b869c30a3a1cbd23d8c2a82773605ec08401b445809ac814ad9f16056f31189> (0)
  *   R Tests/Validation/Wheat/met/MARA1993-1994.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-3c529a066c5f70d3ecc1a6172d2fee11fd24d83e56a9f960d77585723f82ee00> (0)
  *   R Tests/Validation/Wheat/met/MARA1995-1996.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-32d35254d61bae50128858800f6da43a1b6620f16e2ec43f750e1a8da692ea6c> (0)
  *   R Tests/Validation/Wheat/met/MARA1996-1997.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-2b134b4e5af930ff480f980b22b2376e34593c42a97d9d212883d65e16c695e9> (0)
  *   R Tests/Validation/Wheat/met/MARB1992-1993.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-99b5411fb007d2cd40e82c300a780cacdbe1fc0cfe198485aa26da8a8bd989a9> (0)
  *   R Tests/Validation/Wheat/met/MARB1993-1994.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-8c72ba5f61d186ad2e656be155a577d37ec896b3263208172d7b2a265e9a07d1> (0)
  *   R Tests/Validation/Wheat/met/MARB1995-1996.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-561f7374f74a32ef836987d153664aa94b7b179be940ea59f40407d66ce97ce6> (0)
  *   R Tests/Validation/Wheat/met/MARB1996-1997.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-14e0e813ef14bac4dead2dfa0a25fb6e907b789db89f52b890fa73ef7f10d5c6> (0)
  *   R Tests/Validation/Wheat/met/MARC1992-1993.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-cbe39dc22426f929de5ac41893eda9185bcd04d83c87daadaaa2bf5480c71b0f> (0)
  *   R Tests/Validation/Wheat/met/MARC1993-1994.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-7bb85008daa2b2ce6919576f782fc72648c620d6b46dfe9b8f7ed1c9f67e7b3a> (0)
  *   R Tests/Validation/Wheat/met/MARC1995-1996.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-5eb7a950875ae8f4c29e9a03cde3e8b31db949bd5afa08fced16f82423daa70b> (0)
  *   R Tests/Validation/Wheat/met/MARC1996-1997.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ab65982fa5bdd87af5e816c9a314e3d8b67c1173bc14ad7f3d7486c5a3ad431d> (0)
  *   R Tests/Validation/Wheat/met/MER-73.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ac4618499b1d8364dc1d793834970e59376989b6ce0b3c10e93b28df762729c2> (0)
  *   R Tests/Validation/Wheat/met/MER-86.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-2e282f07990bc4f6e8eb7e73a475b49ad10a5c0d93ad89d17b04a2f23e5fa979> (0)
  *   R Tests/Validation/Wheat/met/Maricopa9293.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-0df317c17879c982f43517c7951f07412808d0f394d81c1b1b109d2a0b976d6f> (0)
  *   R Tests/Validation/Wheat/met/Maricopa9296.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-a07eaec7d832fe54205d6a0791be7df858301873830c37bdf11214813bb604c9> (0)
  *   R Tests/Validation/Wheat/met/Maricopa9296B.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-077f46bd956ffeeef896552e05c0079d16a0a05ee6137fe20d70357c570fbe96> (0)
  *   R Tests/Validation/Wheat/met/Maricopa9394.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-fbe5a429b505dbf685db49302c251468c35b69107cc3259efaf8568b371ab728> (0)
  *   R Tests/Validation/Wheat/met/Minnipa2012.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-90fa08de3790ab7dea8e087903495135cad165439dcf70ee459869716a146008> (0)
  *   R Tests/Validation/Wheat/met/Minnipa2015-TOS1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-96ce70db68ea9783ebf63a80b15d672846ff7971b6fbaf3b726f61992480d77d> (0)
  *   R Tests/Validation/Wheat/met/Minnipa2015-TOS2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-bd800c475821cbc184456126b70d4a49c78348b2969832ffb657be5212faabfd> (0)
  *   R Tests/Validation/Wheat/met/Minnipa2015-TOS3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-4256d57215394d2134868daac3b5eb44e8d50b17d1681a40e84bba814b9eb7d3> (0)
  *   R Tests/Validation/Wheat/met/Nangwee2012.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-46c95c45f1c1b77c96b99656ff59fe67e6c132a87f64da8ec79b18941a370622> (0)
  *   R Tests/Validation/Wheat/met/PTOS_10Kairi-1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-30abe382aaa4eeffe97628ed9187426b5b0c6341968c15a0f1616dee6d423194> (0)
  *   R Tests/Validation/Wheat/met/PTOS_10Kairi-2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-b582dcfc2245d1ad5c1e77484745cfcfbb0ddc857ce84c6d1ebed2b052d263b1> (0)
  *   R Tests/Validation/Wheat/met/PTOS_10Kairi-3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-dbed7b50a79ba89d83667835f0315b56c7483cafdcf5fc222a12f3a7d710b2da> (0)
  *   R Tests/Validation/Wheat/met/PTOS_10Kairi-4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-1ff672cc63773620cd2ef0c2e86b7a8e0606c76de02e841a31a461ba0ba3a15a> (0)
  *   R Tests/Validation/Wheat/met/PTOS_10Kairi-5.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-30f0ffa2d026d8100c85b17bb40a2d06193d7d293564498dcafc5dbb3f1594b9> (0)
  *   R Tests/Validation/Wheat/met/PTOS_10Mackay-1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-00e0bff73c4d565abf41c30e5ce70bfbf1ad156381605fc4582250331020dd6c> (0)
  *   R Tests/Validation/Wheat/met/PTOS_10Mackay-2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-15bd336045e29419dea75f47318a1efd588ec7192b3dbb9708db06b1d2419aa4> (0)
  *   R Tests/Validation/Wheat/met/PTOS_10Mackay-3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-332bb06860dcfef04d34084fc98f05709e251ac027b9d3c84c0415289a972de2> (0)
  *   R Tests/Validation/Wheat/met/PTOS_10Mackay-4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-5335cb112fdf4e418e5ccfaee27e39d9ac76e56d5f441d56f28d6aa78ff0bc80> (0)
  *   R Tests/Validation/Wheat/met/PTOS_10Mackay-5.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-7cdea89fb272c24741c97b0c10f378ee291ca0666d7e0eb7596f8dacaf321eb4> (0)
  *   R Tests/Validation/Wheat/met/PalmerstonNorth.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-7b8f1be254b62f59dbd02201b12a17e6acf11d3cb6f8dae34068e67c399c026e> (0)
  *   R Tests/Validation/Wheat/met/RNAT-09Gatton-1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-63cf33fc01d2572698265301f29152ac277376644efa33aca4f89a33f58651c7> (0)
  *   R Tests/Validation/Wheat/met/RNAT-09Gatton-2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-8bb90f36425f495b316a1e968e0dfec1abef7a59bac6047bda35712d9d239107> (0)
  *   R Tests/Validation/Wheat/met/RNAT-09Gatton-3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-3c996b0f218e391021ddba6f94a0f820899990daae63a90dd44f399fd4976623> (0)
  *   R Tests/Validation/Wheat/met/RNAT-09Gatton-4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-b545550bfee0f026bef5e652249bc67ba950fcccabc3bbd7f74d76ba5238130c> (0)
  *   R Tests/Validation/Wheat/met/RNAT-09Gatton-5.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-4447191b6b1779c6ce5dc545682f915d762a12d6f2e46c1bb7e4ee1f4b07d6ab> (0)
  *   R Tests/Validation/Wheat/met/SGEHEAT_13GEHEAT-1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-259ff7d0841a65ade5baf6aa600cf42447ddc0cfdcfdb914b44a87e6c159ff2d> (0)
  *   R Tests/Validation/Wheat/met/SGEHEAT_13GEHEAT-2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-9df9b632ab5f758ce7e2e2eed85ffc3a04f8fa74ac3c2872973d64b9ee89af54> (0)
  *   R Tests/Validation/Wheat/met/SN.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-70239bc2e9d810443b9912e4bb69672566711050351f71fb366920be6514acf4> (0)
  *   R Tests/Validation/Wheat/met/SV.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-42f0671ba196dfdf8c6dbadce1a5c3085d5b37d6f070d9f71b980da4c760d7bf> (0)
  *   R Tests/Validation/Wheat/met/Tamworth1992.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-0a508d1d5930f4338f237f2e430c5658cb1d7c14563b6a71f3c86cc7dbe34b0c> (0)
  *   R Tests/Validation/Wheat/met/Tarlee2011.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-d224385774ab2bf989aa8d75bc62ea4be07c08ab915cfbd21cbd96707c365171> (0)
  *   R Tests/Validation/Wheat/met/Temora2012.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-fd441efe7f4a8fc8f38587374aeca18677808b93af237038b73018f337cce9d4> (0)
  *   R Tests/Validation/Wheat/met/Temora2015-TOS1.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-6378a15282ec421b1a096c3ff2238696beb27af64ac233b06d8b6b19b079c0e7> (0)
  *   R Tests/Validation/Wheat/met/Temora2015-TOS2.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ebbff428d925eccb074a4458219d124ab44f2ea765315c13de4f8712bb35045a> (0)
  *   R Tests/Validation/Wheat/met/Temora2015-TOS3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-55b8c8ea07db0d1d36dbd10c7190a3141ee1ace4670dc66f3cdb40c912f00c29> (0)
  *   R Tests/Validation/Wheat/met/Temora2015-TOS4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-3c5cab2d7abb0c1788029f75e16a976e92a3e19fcdb201436ec216abb6ddcea3> (0)
  *   R Tests/Validation/Wheat/met/Temora2016-TOS4.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-ac1db5c99346495b50564bba4d984de5b0c7258b32d20aa6b6ba52974324f786> (0)
  *   R Tests/Validation/Wheat/met/Wagga1314.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-e117a76fc35541e5c7cd70c3fde71770e59740cedc11b088cc0cef53ad270251> (0)
  *   R Tests/Validation/Wheat/met/Wagga1991.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-2746a880f2b5118ea7dd80fa04e8a0090507634d9a594447ce5b7be6c7b0fdaa> (0)
  *   R Tests/Validation/Wheat/met/WaggaWagga2019.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-2a0b5c1afcd4c38e2d03a3c68fec2ca45e91c15063da7e58d9b19604f418a207> (0)
  *   R Tests/Validation/Wheat/met/WaggaWagga2019NPIfield.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-18fe05944fd30a8d7af2e24602b95822e59c8cf873e6557013783589c0de7bc8> (0)
  *   R Tests/Validation/Wheat/met/Wakanui.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-42bd4ae46c8a31f352d731dededc09cd269616b9685c7ededdf852b632d28efb> (0)
  *   R Tests/Validation/Wheat/met/Walpeup2011.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-f97bb923d1b26c809e422035447bb88d927247e892898ee7ff549f347ff37393> (0)
  *   R Tests/Validation/Wheat/met/Walpeup2012.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-2bc9ff067ed194358aaa0cfb92b55ae8007d070b0eb92224ed034f7df3ae98b0> (0)
  *   R Tests/Validation/Wheat/met/Wongan83.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-6923339033b95b59c96388c6e0e1c7de8238754bb4b85abc47a0dad4ecfedd0f> (0)
  *   R Tests/Validation/Wheat/met/Ycreek.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-6d1ff640fc5fcca19aff73b8dca8cc465bd0e6bb250e21bbd557dc80716ca142> (0)
  *   R Tests/Validation/Wheat/met/Yucheng.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-d2fd186412e62138059e699c6a5b318c799e2bced1a4373bcc1568049767ea07> (0)
  *   R Tests/Validation/Wheat/met/aps02.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-d8967f609b510358fc8ab7a4e8ce263eaf751f8ae5ba69999eaa4373849333c5> (0)
  *   R Tests/Validation/Wheat/met/dale2019NPIfield.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-d65cab78fe3f88e7eff88bcbdfff4f6454330354b49076094de0c55ed4827acd> (0)
  *   R Tests/Validation/Wheat/met/emer96.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-13d29a8d63a0f54b58055b13374ba54a029ecc7556d61248e4c9b9d0ce6d9868> (0)
  *   R Tests/Validation/Wheat/met/gatton-15-g3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-cdc636ac61860837e79010d9e4af865af004dab49fb40e383da1dd1e08e00a6c> (0)
  *   R Tests/Validation/Wheat/met/gatton15g3.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-45e29bb27460d9dcbe3f92dd2a68e6f8804dfd77d04be6590677078d36859869> (0)
  *   R Tests/Validation/Wheat/met/gatton2016.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-2fb525086983ecf76a4d360ac8a9ade1d39bb3283db2b87c13bb0bd6d2ec6230> (0)
  *   R Tests/Validation/Wheat/met/hall2002.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-7d5b26d0c557e8f913baf71f3c70c23e59a587dd69097efa4555ef30dd4f9b25> (0)
  *   R Tests/Validation/Wheat/met/lincoln.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-93134155c0d264574cfb4c6d7fbc619085595530f42897ad8069813a23b9409f> (0)
  *   R Tests/Validation/Wheat/met/miles96.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-bbed3f3fcae2d25d85192d3a3847a20724a79bf69b364f3bc275b6641bdd6c0a> (0)
  *   R Tests/Validation/Wheat/met/moree96.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-630aee7b6df617a82e0a71dc9ec657302b83304359876621ae9e34a26ded1432> (0)
  *   R Tests/Validation/Wheat/met/pendletonAPSIMtav.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-a92e88b610e9e47de9cc7ed2aa4e957095d64a56c7e86db03d3619ad08af0ec0> (0)
  *   R Tests/Validation/Wheat/met/yanyean2019.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-9037c3d55b9dfcdcd38d38db95dec459f16a5685d4baca86104491f8ac09ab19> (0)
  *   R Tests/Validation/Wheat/met/yanyean2019NPIfield.met<https://github.com/APSIMInitiative/ApsimX/pull/9634/files#diff-fbf33c6bff618bb5266cf9463608e2dd3bdd3602fc9ff4b73f721b18d28ba3af> (0)

Patch Links:

  *   https://github.com/APSIMInitiative/ApsimX/pull/9634.patch
  *   https://github.com/APSIMInitiative/ApsimX/pull/9634.diff

—
Reply to this email directly, view it on GitHub<https://github.com/APSIMInitiative/ApsimX/pull/9634>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AC2UVWTBWEG7BCQL3W2HAET2NKT5PAVCNFSM6AAAAABWGKZK2WVHI2DSMVQWIX3LMV43ASLTON2WKOZSHAZDCOJXGU3DAMA>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

HamishBrownPFR commented:
Thanks for the heads up @hut104.  I have been liaising with @par456 to make sure I am not interfering with any of the work he is doing on wheat or what the team are doing on redesigning the test system.  
We need to build an analysis that allows us to look at effects of changes to the wheat model globally and for specific facets of specific tests for the wheat projects we have on the go.  I will prototype something in python scripts and current tidying is to facilitate a clear work flow there.  I will keep the team up to date with how that is going as it might help inform the design of approaches to index, aggregate and review data in the new testing system

HamishBrownPFR commented:
@par456 @hol353 please review and merge if happy.  I have made another commit which removes redundant index columns from data files as these are not used for anything and interfere with the graphs somehow.  I also renamed a number of columns from GS to Wheat.Phenology.Zadok.Stage which is why the stats on this have changed.  I also removed a number of columns of Wheat.Phenology.Zadok.Stage.se which contained only zeros.
